tests/Unit/EnforcerTest.php

```diff
@@ -241,9 +241,9 @@ public function testGetImplicitPermissionsForUser()
 
     public function testGetImplicitUsersForPermission()
     {
-        $e = new Enforcer($this->modelAndPolicyPath . '/rbac_model.conf', $this->modelAndPolicyPath . '/rbac_with_hierarchy_policy.csv');
+        $e = new Enforcer($this->modelAndPolicyPath . '/rbac_model.conf', $this->modelAndPolicyPath . '/rbac_policy.csv');
+
```

Comment:
Why modify the policy file ?

Reply:
This policy should use the old one here.   My fault